tests/codegen-llvm/annotate-moves/annotate-moves-integration.rs

```diff
@@ -0,0 +1,114 @@
+//@ compile-flags: -Z annotate-moves=50 -Copt-level=0 -g
```

Comment:
I'd like to also see tests of various types that follow different configurations than just `struct { array }` - maybe a raw array copy, maybe a copy of other primitives like `u32`, `(u128, u128)` or pointers.

Also interested in how things like `fn foo(x: &) { let _copy = *; }` would look, it's not clear to me that those wouldn't work differently from a stack copy?

Reply:
> Also interested in how things like fn foo(x: &) { let _copy = *; } would look, it's not clear to me that those wouldn't work differently from a stack copy?

I added this and it turns out like regular Copy and is annotated with `compiler_copy`.